internal/collections/map.go

```diff
@@ -162,6 +162,10 @@ func (c *Map) String() string {
 	return res.String()
 }
 
+func (c *Map) Len() int {
```

Comment:
Can you add some small unit tests for `Len` methods?

Reply:
sure